corona-warn-app/cwa-app-android
# Loading Event - Consent I (EXPOSUREAPP-10649)

This PR addresses Exposureapp-10649 (Feature: ExposureApp-10194 (Ticketvalidation)).

**Contains**: 
Entry for the UI / LoadingButton Event Handling.

**How to test:**
- Use cwa cli command: cwa ti gen -o
- Scan QR Code
- Navigate to the Consent 1 Screen
- Press "Einverstanden" on the Consent Screen
(Loading Animation should appear, followed by succesful loading or failure dialog as the Ticketvalidation-Story is in dev state)

Press it
![image](https://user-images.githubusercontent.com/46747780/143253192-dd308f24-735f-48cb-8075-69dd51b8611b.png)
